awsshell/utils.py

```diff
@@ -118,3 +119,16 @@ def file_contents(self, filename, binary=False):
 
     def file_exists(self, filename):
         return filename in self._file_mapping
+
+
+def force_unicode(obj, encoding='utf8'):
```

Comment:
It probably would be nice if you could hook into the botocore response parsers to do this but this looks like the next best thing as there is not really a way to do this.